satpy/readers/ahi_hsd.py

```diff
@@ -280,6 +297,12 @@ def __init__(self, filename, filename_info, filetype_info, mask_space=True):
         self.sensor = 'ahi'
         self.mask_space = mask_space
 
+        calib_mode_choices = ('NOMINAL', 'UPDATE')
```

Comment:
Are these names made up or is this what JMA uses?

Reply:
These names for the himawari calibration are my own, JMA doesn't have a name for what I've called 'nominal', and what I refer to as 'update' they call calibrated coefficients, which (IMO) is a bit confusing. If you prefer I can change the latter to 'calibrated' though.

Reply:
fine by me